scipy/interpolate/__init__.py

```diff
@@ -5,87 +5,115 @@
 
 .. currentmodule:: scipy.interpolate
 
-Sub-package for objects used in interpolation.
+Sub-package for functions and objects used in interpolation.
+
+See the :ref:`user guide <tutorial-interpolate>` for recommendations on choosing a
+routine, and other usage details.
 
-As listed below, this sub-package contains spline functions and classes,
-1-D and multidimensional (univariate and multivariate)
-interpolation classes, Lagrange and Taylor polynomial interpolators, and
-wrappers for `FITPACK <http://www.netlib.org/dierckx/>`__
-and DFITPACK functions.
 
 Univariate interpolation
 ========================
 
 .. autosummary::
    :toctree: generated/
 
-   interp1d
+   make_interp_spline
+   CubicSpline
+   PchipInterpolator
+   Akima1DInterpolator
+   FloaterHormannInterpolator
    BarycentricInterpolator
    KroghInterpolator
-   barycentric_interpolate
-   krogh_interpolate
-   pchip_interpolate
    CubicHermiteSpline
-   PchipInterpolator
-   Akima1DInterpolator
-   CubicSpline
+
+**Low-level data structures for univariate interpolation:**
+
+.. autosummary::
+   :toctree: generated/
+
    PPoly
    BPoly
-   FloaterHormannInterpolator
+   BSpline
 
 
 Multivariate interpolation
 ==========================
 
-Unstructured data:
+**Unstructured data**
 
 .. autosummary::
    :toctree: generated/
 
-   griddata
    LinearNDInterpolator
    NearestNDInterpolator
    CloughTocher2DInterpolator
+   griddata - a convenience wrapper for the above three interpolators
    RBFInterpolator
-   Rbf
    interp2d
 
-For data on a grid:
+**For data on a grid:**
 
 .. autosummary::
    :toctree: generated/
 
-   interpn
    RegularGridInterpolator
-   RectBivariateSpline
+   interpn - a convenience wrapper for `RegularGridInterpolator`
 
 .. seealso::
 
-    `scipy.ndimage.map_coordinates`
+    `scipy.ndimage.map_coordinates`,
+    :ref:`An example wrapper for map_coordinates <tutorial-interpolate_cartesian-grids>`
+
+
+**Low-level data structures for tensor product polynomials and splines:**
 
-Tensor product polynomials:
 
 .. autosummary::
    :toctree: generated/
 
    NdPPoly
    NdBSpline
 
-1-D Splines
-===========
+
+1-D spline smoothing and approximation
+======================================
 
 .. autosummary::
    :toctree: generated/
 
-   BSpline
-   make_interp_spline
    make_lsq_spline
    make_smoothing_spline
    generate_knots
    make_splrep
    make_splprep
 
-Functional interface to FITPACK routines:
+
+Rational Approximation
+======================
+
+.. autosummary::
+   :toctree: generated/
+
+   AAA
+   pade
```

Comment:
Should we de-emphasize this by putting it in additional tools? I think it would belong up here if it had an interface something like the others. If the method has advantages over AAA, it might be worth creating a `Pade` with a similar interface and moving `pade` to the bottom. If not, even more reason to just move `pade` to the bottom.

Reply:
The current implementation of `pade` is flawed so no objections to moving to the bottom. I have a rewrite on my laptop back home that I am planning to contribute but I haven't decided whether to write a new interface or just make it fit the old one . 

`pade` and `AAA` serve different purposes: `AAA` is a bit like fitting a polynomial to data but for rational functions and `pade` is like a rational Taylor series

Reply:
+1 for moving `pade` to additional tools, done. It's going to be an odd one out no matter what, so it's a right place for it indeed. Done.